### DISCUSSION TITLE
Hotswapping settings json

### DISCUSSION THREAD

**Discussion by aujh** [OpenTabletDriver/OpenTabletDriver#1581] [Q&A] [answered]
Was wondering if there is a way to hotswap the settings json from one of the aux buttons, from OTD itself, or is it easier to just run a script through scriptrunner?

**Comment by jamesbt365** [comment, marked as answer]
techincally you can edit the settings file manually but you will still need to reload the settings file by clicking apply or reloading the ux. You may even need to reload the daemon if you edit the settings manually.

**Reply by aujh** [reply]
hmmm okay, sounds like a bit of a trek. thanks for the help :)

**Comment by gonX** [comment]
I suggest using the CLI for this with the `load` and `save` commands, and you can indeed use ScriptRunner for this.

If you want this as an easier to use binding, the next major release containing #2772 includes being able to set presets with bindings.
